pkg/controller/vitesskeyspace/reconcile_resharding.go

```diff
@@ -136,3 +156,33 @@ func (r *reconcileHandler) reconcileResharding(ctx context.Context) (reconcile.R
 
 	return resultBuilder.Result()
 }
+
+func (r reconcileHandler) shardsRowCount(ctx context.Context, shardNames []string) (uint64, error) {
+	var rowCount uint64
+	for _, shardName := range shardNames {
+		tabletMap, err := r.ts.GetTabletMapForShard(ctx, r.vtk.Spec.Name, shardName)
+		if err != nil {
+			return 0, err
```

Comment:
Let's include the loop var in all errors so we know which one failed:

```suggestion
			return 0, fmt.Errorf("failed to get tablets for shard %v: %v", shardName, err)
```

Reply:
Done.